extensions/wikia/WikiFactory/Hubs/WikiFactoryHub.php

```diff
@@ -175,6 +175,10 @@ public function getCategoryId( $city_id ) {
 	 */
 
 	public function getVerticalId( $city_id ) {
+		global $wgWikiaEnvironment;
+		if ( $wgWikiaEnvironment == WIKIA_ENV_INTERNAL ) {
+			$city_id = 11;
```

Comment:
To be honest. I wanted to put it in config, but... 
... tons of code understands that when ```$wgCityId``` is ```null``` we are on Wikia One. This way I just C&P from https://github.com/Wikia/app/blob/CON-2383/extensions/wikia/WikiFactory/Hubs/WikiFactoryHub.php#L204
Do you think moving ```11``` into constant is enough?